File: MyDailyBeat/MyDailyBeat/EVCLabelTableViewCell.m
//
//  EVCLabelTableViewCell.m
//  MyDailyBeat
//
//  Created by Virinchi Balabhadrapatruni on 10/18/14.
//  Copyright (c) 2014 eVerveCorp. All rights reserved.
//

#import "EVCLabelTableViewCell.h"

@implementation EVCLabelTableViewCell

@synthesize messageLbl;
- (id) initWithMessage:(NSString *)lblText {
    NSArray *nib = [[NSBundle mainBundle] loadNibNamed:@"EVCLabelTableViewCell_iPhone" owner:self options:nil];
    self = [nib objectAtIndex:0];
    if (self) {
        self.message = lblText;
    }
    
    return self;
}

- (void)awakeFromNib {
    [super awakeFromNib];
    self.messageLbl.text = self.message;
}

@end

File: MDB_Swift/MyDailyBeat/MyDailyBeat/src/EVCLabelTableViewCell.swift

//
//  EVCLabelTableViewCell.swift
//  MyDailyBeat
//
//  Created by Virinchi Balabhadrapatruni on 10/18/14.
//  Copyright (c) 2014 eVerveCorp. All rights reserved.
//
import UIKit
class EVCLabelTableViewCell: UITableViewCell {
    @IBOutlet var messageLbl: UILabel!
    var message: String = ""

    override func awakeFromNib() {
        super.awakeFromNib()
        self.messageLbl.text = self.message
    }
}
